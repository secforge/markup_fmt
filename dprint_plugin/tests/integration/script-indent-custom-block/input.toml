# Test that script indent works correctly in custom blocks with lang attribute
scriptIndent = true
printWidth = 80

[vue_custom_block]
default = "lang-attribute"
