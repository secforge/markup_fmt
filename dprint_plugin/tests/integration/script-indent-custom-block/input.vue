<template>
  <div>Test script indent in custom blocks</div>
</template>

<!-- Custom block with lang="javascript" and scriptIndent enabled -->
<i18n lang="javascript">
const messages = { hello: "Hello", goodbye: "Goodbye", welcome: "Welcome to our application" };
</i18n>

<script>
const messages = { hello: "Hello", goodbye: "Goodbye", welcome: "Welcome to our application" };
</script>
