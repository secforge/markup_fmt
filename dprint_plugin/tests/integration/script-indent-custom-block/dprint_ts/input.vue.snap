---
source: dprint_plugin/tests/integration.rs
assertion_line: 123
---
<template>
  <div>Test script indent in custom blocks</div>
</template>

<!-- Custom block with lang="javascript" and scriptIndent enabled -->
<i18n lang="javascript">
  const messages = {
    hello: "Hello",
    goodbye: "Goodbye",
    welcome: "Welcome to our application",
  };
</i18n>

<script>
  const messages = {
    hello: "Hello",
    goodbye: "Goodbye",
    welcome: "Welcome to our application",
  };
</script>
